### PR TITLE
Logs

### PR DESCRIPTION
**Pull request by Jhoysbou**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-  ]
+    "overrides": [
+        {
+            "files": [
+                "*.js",
+                "*.ts",
+                "*.tsx"
+            ],
+            "options": {
+                "trailingComma": "all",
```
Comment (Jhoysbou) [inline .prettierrc:10]: Switched to `all` for trailing commas. Are you ok with this?

File: packages/messageProxy/src/doc/doc.service.ts [hunk 2]
```diff
@@ -37,60 +38,67 @@ export class DocService implements OnModuleDestroy {
...
     }
 
-    syncComplete(connection: WebSocketConnection, payload: SyncCompletePayload) {
-        this.actorService.syncComplete(connection.id, payload);
+    syncComplete(conn: WebSocketConnection, payload: SyncCompletePayload) {
+        this.actorService.syncComplete(conn.id, payload);
     }
 
-    joinToDoc(connection: WebSocketConnection, docId: DocKey): JoinResponsePayload {
+    joinToDoc(conn: WebSocketConnection, docId: DocKey): JoinResponsePayload {
```
Comment (evilfant) [inline packages/messageProxy/src/doc/doc.service.ts:101]: it's nothing, but why do you prefer to rename connection -> conn ?

File: packages/docHandler/package.json [hunk 3]
```diff
@@ -1,48 +1,49 @@
...
+        "publish:image": "./scripts/dockerPublish.sh",
+        "prepare": "npm run build:lib",
+        "start": "node dist/main.js",
+        "start:dev": "nodemon --ext \".ts,.js\" --watch \"./src\" --exec \"ts-node ./src/main.ts\" & yarn build:lib:dev",
+        "clear": "rimraf dist lib",
+        "start:dapr": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev yarn start",
+        "start:dapr:test": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev node test",
+        "start:dapr:dev": "dapr run --app-id docHandler --app-protocol http --app-port 3001 --dapr-http-port 3501 --config ./dapr-config.yaml --placement-host-address localhost:50006 --resources-path ../../daprComponents/dev yarn start:dev"
+    },
+    "dependencies": {
```
Comment (evilfant) [inline packages/docHandler/package.json:33]: in json must be 2 spaces tab
it's in prettier config and npm uses 2 spaces. If to change to 4 npm will return it to 2 after next manipulation with package.json

**Review by evilfant — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 1 above)
Comment (evilfant) [reply .prettierrc:10]: yeap, this is fine :+1:

File: packages/messageProxy/src/doc/doc.service.ts (hunk 2 above)
Comment (Jhoysbou) [reply packages/messageProxy/src/doc/doc.service.ts:101]: `connection` is too long

I think `conn` is pretty straightforward variable name.
If you don't like it, no big deal, I can rename it

File: packages/docHandler/package.json (hunk 3 above)
Comment (Jhoysbou) [reply packages/docHandler/package.json:33]: Yeah, I guess I need to change my editor settings. I'll come back with a patch

**Comment by Jhoysbou** [comment]
Formatted the whole project with prettier

**Review by evilfant — APPROVED** [review]

**Pull request merged**